library/alloc/src/vec.rs

```diff
@@ -3026,7 +3026,10 @@ impl<T> Drain<'_, T> {
     }
 }
 
-/// An iterator produced by calling `drain_filter` on Vec.
+/// A draining iterator with filter predicate for `Vec`.
+///
+/// This struct is created by [`Vec::drain_filter()`].
```

Comment:
I don't quite like the brackets.
```suggestion
/// This struct is created by [`Vec::drain_filter`].
```